config/development.json

```diff
@@ -170,6 +170,7 @@
 		"onboarding/trail-map-feature-grid-structure": false,
 		"onboarding/trail-map-feature-grid": false,
 		"onboarding/user-on-stepper-hosting": true,
+		"onboarding/goals-first": true,
```

Comment:
hmmm...it seems the default value should be false on both dev and staginge since it is far from complete. Or we should limit it to the `onboarding` flow and keep  the site-setup flow the same without the config

Reply:
Proposed https://github.com/Automattic/wp-calypso/pull/97219